skxray/core/accumulators/histogram.pyx

```diff
@@ -184,31 +184,32 @@ class Histogram:
         numtypeindex = {tp : i for i, tp in enumerate(numtypes)}
         ctypeindices = [numtypeindex.get(x.dtype, 999) for x in coords]
         coordsorder = np.argsort(ctypeindices, kind='mergesort')
-        istrides = np.asarray(self._values.strides, dtype=np.int32)[coordsorder]
+        istrides = np.asarray(self._values.strides)[coordsorder]
         istrides //= self._values.itemsize
-        cdef int [:] dataindexstrides = istrides
+        cdef int dataindexstrides[MAX_DIMENSIONS]
+        cdef int i
+        for i in range(self.ndims):
```

Comment:
is this `i` index the one that is being defined above on line `190`?  I have not looked at the C code to see if it is actually using this variable.